Development branch

kiregor opened the pull request:
Most of the refactors requested this morning, with some minor code updates.

rameez-j committed:
Added css for listings page

HamzzAR committed:
Merge remote-tracking branch 'origin/development-branch' into
pay-for-booking-integration

# Conflicts:
#	src/main/java/com/qa/CinemaProject/service/BookingService.java

rameezjhaveri reviewed (approved):
Nice Work!

frobian21 committed:
added unregistered phone number

frobian21 committed:
Merge pull request #28 from kiregor/pay-for-booking-integration

Pay for booking integration

Commit:
selenium test on movie page added

rameezjhaveri reviewed (dismissed):
I approve of your color choices, nice work!

kiregor committed:
Merge pull request #29 from kiregor/colour-changes

colour changes

rameez-j committed:
Added error page for broken/not existing links

HamzzAR committed:
Booking successful page created. Whenever user pays for his bookings
successfully, they would be redirected to the success page along with
the summary of their tickets

rameez-j committed:
added age rating api fetch

rameez-j committed:
Ammended fetch for age ratings, changed image link to a different url

Commit:
selenium tests improved

Commit:
Merge https://github.com/kiregor/CinemaProject into selenium-branch

rameez-j committed:
Refactored age rating fetch request

frobian21 committed:
added button focus colours to navbar

rameez-j committed:
Resolved merge conflicts

rameez-j committed:
update: Resolved merge conflicts

LindaAllyket committed:
Merge pull request #31 from kiregor/colour-changes

Colour changes

frobian21 committed:
Merge pull request #33 from kiregor/pagination-branch

Pagination branch

HamzzAR committed:
implemented transaction decline status

HamzzAR committed:
Merge remote-tracking branch 'origin/development-branch' into
pay-for-booking-integration

# Conflicts:
#	Client/src/components/AppPages.js

phtb233 committed:
Merge pull request #34 from kiregor/pay-for-booking-integration

Pay for booking integration

phtb233 committed:
got the embedded mongodb using sequences

rameez-j committed:
Fixed bug for carousel unlimeted loading

rameez-j committed:
Resolved merge conflicts

rameez-j committed:
.

frobian21 committed:
Merge branch 'development-branch' into selenium-branch

frobian21 committed:
Merge pull request #35 from kiregor/pagination-branch

Fixed bug for carousel unlimeted loading

frobian21 committed:
Merge pull request #32 from kiregor/selenium-branch

Selenium branch

HamzzAR committed:
If card has insufficient funds, it will show a toast message

frobian21 committed:
added dependency to package

frobian21 committed:
Merge pull request #37 from kiregor/pay-for-booking-integration

If card has insufficient funds, it will show a toast message

HamzzAR committed:
Merge pull request #38 from kiregor/package-update

added dependency to package

HamzzAR committed:
Improved booking success page and insufficient fund message improved

phtb233 committed:
resolved merge conflicts, fixed bread crumbs

phtb233 committed:
breadcrumbs capitalization

rameez-j committed:
Implemented stickey navbar and footer

phtb233 committed:
still writing embedded mongodb tests

kiregor committed:
All selenium tests made so far are working

phtb233 committed:
minor changes

phtb233 committed:
Merge pull request #39 from kiregor/selenium-branch

All selenium tests made so far are working

rameez-j committed:
Justified footer with flex

HamzzAR committed:
Card details validation implemented, shows a pop-up message when card
details are not entered

frobian21 committed:
Merge pull request #40 from kiregor/pay-for-booking-integration

Pay for booking integration

phtb233 committed:
found incorrect parameter in MovieService::updateMovie

phtb233 committed:
using EclEmma to increase coverage

rameez-j committed:
Added page flip on mouse over to current lisings page

phtb233 committed:
created dummy repos/services

phtb233 committed:
unit tests for entities, services

rameez-j committed:
Added buttons to current listings page to go directly to booking or more info

rameez-j committed:
Added working link and margins to button

kiregor committed:
Resolved merge conflicts

phtb233 committed:
removed api-movie import

phtb233 committed:
removed apimovie comment

rameez-j committed:
Added small carousel to homepage for new releases

HamzzAR committed:
Merge pull request #41 from kiregor/test-mockito

Test mockito

HamzzAR committed:
Checkout form improved. Now has name and email.

kiregor committed:
Finished implementing code to complete bookings with the Seats.io api

kiregor committed:
front end addition

kiregor committed:
checking up to date

kiregor committed:
checking up to date

kiregor committed:
Merge remote-tracking branch 'origin/development-branch' into
event-branch

# Conflicts:
#	Client/src/components/Seating/AppSeatingPage.js
#	src/main/java/com/qa/CinemaProject/constants/MappingConstants.java
#	src/main/java/com/qa/CinemaProject/service/BookingService.java

frobian21 committed:
footer has been fixed

frobian21 committed:
Merge pull request #43 from kiregor/event-branch

Event branch

phtb233 committed:
started search bar functionality

phtb233 committed:
resolved merge issues

frobian21 committed:
reduced breadcrumb size and fixed sticky footer and header

kiregor committed:
Added movie update functionality to the back end

Commit:
Movie page tweeked and tests improved

frobian21 committed:
changed alignment of book-now and more info buttons#

phtb233 committed:
Merge pull request #46 from kiregor/pay-for-booking-integration

Checkout form improved. Now has name and email.

kiregor committed:
Merge pull request #45 from kiregor/selenium-branch

Selenium branch

phtb233 committed:
things

HamzzAR committed:
Merge pull request #44 from kiregor/aesthetics-branch

Aesthetics branch

HamzzAR committed:
Google Sign-in in progress

phtb233 committed:
Merge branch 'development-branch' of
https://github.com/kiregor/CinemaProject into development-branch

# Conflicts:
#	Client/src/components/AppPages.css

frobian21 committed:
Merge pull request #47 from kiregor/search-bar-development

Search bar development

rameez-j committed:
Removed small carousel

Commit:
colour theme implemented

Commit:
selenium test on booking feature part 1

rameez-j committed:
moved buttons inline

phtb233 committed:
implemented search function, breadcrumbs

phtb233 committed:
resolved merge conflicts

phtb233 committed:
changed searchbar so it pulls from the backend

rameez-j committed:
Refactored current listings and future listings into 1  document -- had too much repeated code.

Commit:
stuff

kiregor committed:
Added backend functionality for creating new events at the cinema

rameez-j committed:
changed fetch for homepage carousel to tmdb api

Commit:
Merge branch 'development-branch' into selenium-branch

# Conflicts:
#	Client/src/components/Current Listings/ApiMovie.js

kiregor committed:
resolved merge conflicts

Commit:
Merge branch 'selenium-branch' of
https://github.com/kiregor/CinemaProject.git into selenium-branch

Conflicts:
	Client/src/components/Current Listings/ApiMovie2.css
	Client/src/components/Current Listings/ApiMovie2.js

frobian21 committed:
Merge pull request #49 from kiregor/event-branch

Event branch

kiregor committed:
Resolved a merge conflict

phtb233 committed:
admin login form stubbed out

phtb233 committed:
Merge branch 'admin-frontend' of https://github.com/kiregor/CinemaProject into admin-frontend

phtb233 committed:
stubbed out communication with backend

phtb233 committed:
Merge branch 'event-branch' into admin-frontend

kiregor committed:
Added basic backend support for admin login

Commit:
commit

HamzzAR committed:
Implemented Login with google API

rameezjhaveri committed:
Merge pull request #52 from kiregor/selenium-branch

Selenium branch

frobian21 committed:
Merge pull request #51 from kiregor/event-branch

Resolved a merge conflict

frobian21 committed:
Making Login page

HamzzAR committed:
Navbar cleaned up

HamzzAR committed:
NavBar cleaned up

HamzzAR committed:
Close button color changed

frobian21 committed:
moved colours to constants file

rameez-j committed:
Implementing small homepage carousels for new releases

frobian21 committed:
Refactor of sitewide colours

HamzzAR committed:
Merge remote-tracking branch 'origin/development-branch' into
login-with-google

# Conflicts:
#	Client/package-lock.json
#	Client/package.json
#	Client/src/components/Header/AppNavbar.js

frobian21 committed:
refactor to sitewide colours

phtb233 committed:
Merge pull request #53 from kiregor/login-with-google

Login with google

rameez-j committed:
Created new release carousel for homepage

frobian21 committed:
Added userID to booking

frobian21 committed:
Merge branch 'development-branch' of https://github.com/kiregor/CinemaProject into development-branch

frobian21 committed:
login service

Commit:
tests fixed/varying xpath issues

kiregor committed:
test on booking nearly done

Commit:
java version in pom xml 1.9

phtb233 committed:
event-page admin control stubbed out

rameez-j committed:
added second carousel for new releases on homepage

HamzzAR committed:
Implementation done for getting all bookings for a specific user

Commit:
pom xml

Commit:
Merge branch 'selenium-branch' of https://github.com/kiregor/CinemaProject.git into selenium-branch

Commit:
war packaging to get rid of pom xml error

phtb233 committed:
event page admin control fully implemented

phtb233 committed:
Merge branch 'development-branch' into admin-frontend

Commit:
removed release number from pom

HamzzAR committed:
Accounts page in progress

HamzzAR committed:
Merge remote-tracking branch 'origin/development-branch' into login-page

rameez-j committed:
Improved now playing carousel on homepage

Commit:
updated release version

Commit:
removed release info

Commit:
plugin params for maven for java 11

Commit:
Ignoring the warnings

rameez-j committed:
Added carousel for future releases on hompepage with working api call

rameez-j committed:
Fixed merge conflicts

phtb233 committed:
added search bar

phtb233 committed:
fixed breadcrumbs

phtb233 committed:
breadcrumb mistake

phtb233 committed:
Merge pull request #55 from kiregor/aesthetics-branch

Aesthetics branch

HamzzAR committed:
Merge pull request #57 from kiregor/search-bar-development

Search bar development

phtb233 committed:
Merge pull request #54 from kiregor/selenium-branch

Tests fixed/varying xpath issues/ few id's added

Commit:
this

rameez-j committed:
Carousel css fix

Commit:
this

Commit:
added modules removed from jre after java 8

Commit:
commmitted

Commit:
updated pom again

phtb233 committed:
implemented getAllEvents in backend

rameez-j committed:
Added post request to grab movie timings for movie page

HamzzAR committed:
Logout functionality completed.

phtb233 committed:
got event backend working

phtb233 committed:
Merge branch 'development-branch' into admin-frontend

rameez-j committed:
added dependencies and inserted logo into repo

HamzzAR committed:
getting user bookings in progress

rameezjhaveri committed:
Merge pull request #58 from kiregor/admin-frontend

Admin frontend

rameez-j committed:
Test merge/commit

rameez-j committed:
Changes to logo on app navbar

phtb233 committed:
commented out clashing unit test

HamzzAR committed:
Getting user bookings to his account page in progress

Commit:
Places nearby quick link added

Commit:
"Places nearby added"

Commit:
commit

phtb233 committed:
link correct user name/password to event-page

phtb233 committed:
Merge branch 'admin-frontend' of https://github.com/kiregor/CinemaProject into admin-frontend

Commit:
merge conflicts

Commit:
merge conflict

frobian21 committed:
changes

frobian21 committed:
merge conflicts

frobian21 committed:
merge conflicts with dev

frobian21 committed:
package dependencies

Commit:
Places to go added

Commit:
merge conflicts solved

frobian21 committed:
changed account foulder name

frobian21 committed:
changed Summary folder name

frobian21 committed:
summary folder changed name

Commit:
merged with login-page branch

Commit:
deleted Futuremovie.js

Commit:
improved current listings layout

kiregor committed:
Working post to retrieve movie evetns

phtb233 committed:
fixed login buttons/ added credential check

kiregor committed:
Working times on movie page

kiregor committed:
Passing key to seats page

kiregor committed:
Successfully accessed the correct event in seats.io

phtb233 committed:
Merge pull request #61 from kiregor/event-frontend-branch

Event frontend branch

rameez-j committed:
Logo on navbar  - visible

phtb233 committed:
changed application.properties admin credentials

HamzzAR committed:
My account functionality and font-end page completed

phtb233 committed:
Merge branch 'development-branch' into admin-frontend

frobian21 committed:
summary formatting

rameez-j committed:
Fixed bug where breadcrumbs on listings page was redirecting to the wrong page

HamzzAR committed:
Fix bugs and clean layout

frobian21 committed:
merge conflicts

Commit:
screens page created

Commit:
Screens page added and merged to dev

HamzzAR committed:
Merge remote-tracking branch 'origin/development-branch' into login-page

# Conflicts:
#	Client/src/components/MoviePage/ApiMovieDisplay.js
#	src/main/java/com/qa/CinemaProject/controllers/MovieController.java

rameezjhaveri committed:
Merge pull request #64 from kiregor/login-page

Login page

Commit:
merge conflicts

phtb233 committed:
Merge branch 'development-branch' into admin-frontend

phtb233 committed:
Merge pull request #63 from kiregor/admin-frontend

Admin frontend

LindaAllyket committed:
Merge pull request #65 from kiregor/placesnearby-branch

Added screens page

frobian21 committed:
colour changes

frobian21 committed:
merge conflicts

frobian21 committed:
merge conflicts

rameez-j committed:
Added ageRating to movie pages

frobian21 committed:
removed future listings error from merge conflict

frobian21 committed:
changed folder name for summary and accounts

Commit:
screen page edited

Commit:
commit

rameez-j committed:
Added BBFC icons to age ratings

rameez-j committed:
resolved merge conflicts

Commit:
merge

LindaAllyket committed:
Merge pull request #66 from kiregor/login-page

Login page

frobian21 committed:
Merge pull request #68 from kiregor/ageRatings-branch

Age ratings branch

frobian21 committed:
Merge pull request #67 from kiregor/placesnearby-branch

Screens page

rameez-j committed:
Fixed bug where listings breadcrumbs would redirect to incorrect page

rameez-j committed:
final edits - logo, button colors

rameez-j committed:
FInal aesthetics shanges

rameez-j committed:
Updated about us page

phtb233 committed:
fixed userId in booking entity

HamzzAR committed:
Merge pull request #70 from kiregor/bug-fixes

fixed userId in booking entity

kiregor committed:
Merge pull request #69 from kiregor/aesthetics-branch

Aesthetics branch

phtb233 committed:
fixed minor issues

frobian21 committed:
changed colour of payment button

rameez-j committed:
tried to make entire button a link

rameez-j committed:
merged

frobian21 committed:
limited colour change of text on screenspage

LindaAllyket committed:
Merge pull request #71 from kiregor/aesthetics-branch

changed colour of payment button

rameez-j committed:
refactoring code + bug fix for pagination

rameez-j committed:
More refactoring and removing console logs used for debugging

HamzzAR committed:
Merge pull request #72 from kiregor/pagination-branch

Pagination branch

phtb233 committed:
removed some warnings, added admin page alerts

kiregor committed:
Added seperate admin backend

kiregor committed:
fully working seperate admin backend

kiregor committed:
minor cosmetic changes

kiregor committed:
Added new screen button

rameez-j committed:
made carousel go reverse

rameez-j committed:
removed console logs

phtb233 committed:
added notification to event screen

rameez-j committed:
Resolved merge conflicts

phtb233 committed:
resolved conflicts, fixed booking id for guests

phtb233 committed:
finished admin backend

phtb233 committed:
resolved conflict

phtb233 committed:
Merge branch 'development-branch' into admin-branch

rameez-j committed:
Code smells - fixing

rameez-j committed:
Solved merge conflicts

rameez-j committed:
removed unused variables

phtb233 committed:
Merge pull request #73 from kiregor/aesthetics-branch

Aesthetics branch

phtb233 committed:
fixed object assign error and logo case

kiregor committed:
Merge pull request #75 from kiregor/fix-object-width-error

fixed object assign error and logo case
